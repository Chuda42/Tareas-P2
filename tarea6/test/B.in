hacerVecinos 1 2 1.0
sonVecinos 1 2
sonVecinos 3 4
distancia 1 2
vecinos 1
vecinos 2
vecinos 4
hayMParejas
hacerVecinos 1 4 1.0
hacerVecinos 3 2 1.0
vecinos 1
vecinos 2
vecinos 4
hayMParejas
asociarEnMap 5 5.0
asociarEnMap 15 5.0
insertarEnCP 4 4.0
insertarEnCP 9 9.0
insertarEnCP 3 3.0
valorEnMap 5
valorEnMap 15
desasociarEnMap 5
valorEnMap 15
prioritario
actualizarEnCP 3 30.0
prioritario
eliminarPrioritario
estaVaciaCP
Fin
