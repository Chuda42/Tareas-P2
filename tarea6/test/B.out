1>Nuevos vecinos: 1 2 1.00.
2>Son vecinos.
3>NO son vecinos.
4>1.00
5>2 
6>1 
7>
8>NO hay M parejas.
9>Nuevos vecinos: 1 4 1.00.
10>Nuevos vecinos: 3 2 1.00.
11>2 4 
12>1 3 
13>1 
14>NO hay M parejas.
15>Establecida la asociación.
16>Establecida la asociación.
17>Se insertó el elemento.
18>Se insertó el elemento.
19>Se insertó el elemento.
20>5.00
21>5.00
22>Eliminada la asociación.
23>5.00
24>3
25>Se actualizó el valor.
26>4
27>Eliminado.
28>No vacía.
29>Fin.
